Sasquatch/Sasquatch.xcodeproj/project.pbxproj

```diff
@@ -2681,9 +2381,9 @@
 				CODE_SIGN_ENTITLEMENTS = SasquatchObjC/SasquatchObjC.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = "";
```

Comment:
Whoops, please remove all `akvelon` settings :)

Reply:
@clpolet, it's my bad, I will fix it )